internal/driver/driver.go

```diff
@@ -78,19 +81,34 @@ func (linodeDriver *LinodeDriver) SetupLinodeDriver(
 	linodeDriver.name = name
 	linodeDriver.vendorVersion = vendorVersion
 
-	matched, err := regexp.MatchString(`^[0-9A-Za-z_-]{0,`+strconv.Itoa(linodeBSPrefixLength)+`}$`, bsPrefix)
+	// Validate the volume label prefix, if it is set.
+	//
+	// First, we want to make sure it is the right length, then we will make
+	// sure it only contains the acceptable characters.
+	//
+	// When checking the length, we will convert it to a []rune first, to count
+	// the number of unicode characters.
+	if r := []rune(bsPrefix); len(r) > MaxVolumeLabelPrefixLength {
```

Comment:
Question: Do we want to keep the 'bsPrefix' around or rename it?

Reply:
We could rename it. :smile:

Reply:
Renamed!